tests/apps/secretapp/app.go

```diff
@@ -129,18 +136,21 @@ func handler(w http.ResponseWriter, r *http.Request) {
 	cmd := mux.Vars(r)["command"]
 	switch cmd {
 	case "get":
-		secrets, err = getAll(req.Secrets)
+		secrets, statusCode, err = getAll(req.Secrets)
 		res.Secrets = secrets
+		if statusCode != 200 {
+			res.Message = err.Error()
+		}
 	default:
 		err = fmt.Errorf("invalid URI: %s", uri)
 		statusCode = http.StatusBadRequest
 		res.Message = err.Error()
 	}
 
-	if err != nil && statusCode == http.StatusOK {
-		statusCode = http.StatusInternalServerError
-		res.Message = err.Error()
-	}
+	// if err != nil && statusCode == http.StatusOK {
+	// 	statusCode = http.StatusInternalServerError
+	// 	res.Message = err.Error()
+	// }
```

Comment:
Is this intentional or left-over?

Please document if intentional.

Reply:
good catch ... removed.